packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -1339,9 +1335,9 @@ class _TableRowState<T> extends State<TableRow<T>>
   }
 
   /// Presents the content of this row.
-  Widget tableRowFor(BuildContext context, {VoidCallback onPressed}) {
-    Widget columnFor(ColumnData<T> column, double columnWidth) {
-      Widget content;
+  Widget tableRowFor(BuildContext context, {VoidCallback? onPressed}) {
+    Widget columnFor(ColumnData<T?> column, double columnWidth) {
+      Widget? content;
```

Comment:
this can be late

Reply:
nope. it can be assigned multiple times below, under 'else'

Reply:
right, it can't be `late final`, but it can be `late`. If the one call to `??=` is giving an error, just add an else statement to `if (column is ColumnRenderer)` and assign `content` without the `??=` in the else statement.